app-modules/etno/src/Jobs/ProcessItemMediaUpload.php

```
@@ -0,0 +1,41 @@
+    public function __construct(
+        public Item $item,
+        public string $filePath,
+        public string $originalName,
+        public string $mimeType,
+        public array $customProperties = []
+    ) {}
+
+    public function handle(ItemRepository $itemRepository): void
+    {
+        try {
+            $collection = Item::getMediaCollectionName($this->mimeType) ?? throw new \InvalidArgumentException("Unsupported mime type: {$this->mimeType}");
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Metafori\Etno\Models\Item' in method 'handle'. (undefined)

(StaticAccess)